sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -117,62 +69,113 @@ cmake -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD="X86" \
 -DLLVM_EXTERNAL_SYCL_SOURCE_DIR=$SYCL_HOME/llvm/sycl \
 -DLLVM_EXTERNAL_LLVM_SPIRV_SOURCE_DIR=$SYCL_HOME/llvm/llvm-spirv \
--DOpenCL_INCLUDE_DIR="$OPENCL_HEADERS" -DOpenCL_LIBRARY="$ICD_LIB" \
 $SYCL_HOME/llvm/llvm
 
 make -j`nproc` sycl-toolchain
 ```
-### Windows:
-```bash
-mkdir %SYCL_HOME%
+
+**Windows**
+```bat
 cd %SYCL_HOME%
 git clone https://github.com/intel/llvm -b sycl
 mkdir %SYCL_HOME%\build
 cd %SYCL_HOME%\build
 
-cmake -G "Ninja" -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD="X86" -DLLVM_EXTERNAL_PROJECTS="llvm-spirv;sycl" -DLLVM_ENABLE_PROJECTS="clang;llvm-spirv;sycl" -DLLVM_EXTERNAL_SYCL_SOURCE_DIR="%SYCL_HOME%\llvm\sycl" -DLLVM_EXTERNAL_LLVM_SPIRV_SOURCE_DIR="%SYCL_HOME%\llvm\llvm-spirv" -DCMAKE_C_COMPILER=cl -DCMAKE_CXX_COMPILER=cl -DCMAKE_C_FLAGS="/GS" -DCMAKE_CXX_FLAGS="/GS" -DCMAKE_EXE_LINKER_FLAGS="/NXCompat /DynamicBase" -DCMAKE_SHARED_LINKER_FLAGS="/NXCompat /DynamicBase" -DOpenCL_INCLUDE_DIR="%OPENCL_HEADERS%" -DOpenCL_LIBRARY="%ICD_LIB%" "%SYCL_HOME%\llvm\llvm"
+cmake -G "Ninja" -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD="X86"^
+-DLLVM_EXTERNAL_PROJECTS="llvm-spirv;sycl"^
+-DLLVM_ENABLE_PROJECTS="clang;llvm-spirv;sycl"^
+-DLLVM_EXTERNAL_SYCL_SOURCE_DIR="%SYCL_HOME%\llvm\sycl"^
+-DLLVM_EXTERNAL_LLVM_SPIRV_SOURCE_DIR="%SYCL_HOME%\llvm\llvm-spirv"^
+-DCMAKE_C_COMPILER=cl -DCMAKE_CXX_COMPILER=cl -DCMAKE_C_FLAGS="/GS"^
+-DCMAKE_CXX_FLAGS="/GS" -DCMAKE_EXE_LINKER_FLAGS="/NXCompat /DynamicBase"^
+-DCMAKE_SHARED_LINKER_FLAGS="/NXCompat /DynamicBase"^
+"%SYCL_HOME%\llvm\llvm"
 
 ninja sycl-toolchain
 ```
 
-After the build completed, the SYCL compiler/include/libraries can be found
-in `$SYCL_HOME/build` directory.
+TODO: add instructions how to deploy built SYCL toolchain.
 
----
-
-## Build the SYCL runtime with libc++ library.
+## Build SYCL toolchain with libc++ library
 
 There is experimental support for building and linking SYCL runtime with
-libc++ library instead of libstdc++. To enable it the following cmake options
+libc++ library instead of libstdc++. To enable it the following CMake options
 should be used.
-### Linux:
+
+**Linux**
 ```
 -DSYCL_USE_LIBCXX=ON \
 -DSYCL_LIBCXX_INCLUDE_PATH=<path to libc++ headers> \
 -DSYCL_LIBCXX_LIBRARY_PATH=<path to libc++ and libc++abi libraries>
 ```
-# Test the SYCL compiler and runtime
 
-After building the SYCL compiler and runtime, choose the amount of LIT testing you need and run one of the LIT tests suites shown below:
-### Linux:
+# Use SYCL toolchain
+
+## Install low level runtime
+
+To run SYCL  applications on OpenCL devices, OpenCL implementation(s) must be
+present in the system.
+
+To execute SYCL tests on Intel devices following runtimes should be installed:
+
+**Linux**
+
+* Intel GPU [Intel&reg; Graphics Compute Runtime for
+   OpenCL&trade;](https://github.com/intel/compute-runtime/releases)
+* Intel CPU - [Experimental Intel&reg; CPU Runtime for OpenCL&trade; Applications
+   with SYCL support](https://github.com/intel/llvm/releases/tag/2019-08)
```

Comment:
for GPU you passed the general location for CPU RT you passed exact version.
Also different approach is used on Windows and Linux. Can we have common was of specifying RTs.

Reply:
I think that is okay to put link to a general location here. Certain recommended version is changed from time to time and mentioned in release notes. So, probably it worth say that exact recommended version can be found in release notes

Reply:
> Also different approach is used on Windows and Linux. Can we have common was of specifying RTs.

What do you mean by "different approach"? Could you clarify, please?